tests/unit_tests/test_writer_flashlfq.py

```diff
@@ -6,21 +6,6 @@
 import pandas as pd
 
 
-def test_sanity(psms, tmp_path):
-    """Run simple sanity checks"""
-    conf = psms.assign_confidence()
-    test1 = conf.to_flashlfq(tmp_path / "test1.txt")
-    mokapot.to_flashlfq(conf, tmp_path / "test2.txt")
-    test3 = mokapot.to_flashlfq([conf, conf], tmp_path / "test3.txt")
-    with pytest.raises(ValueError):
-        mokapot.to_flashlfq("blah", tmp_path / "test4.txt")
-
-    df1 = pd.read_table(test1)
-    df3 = pd.read_table(test3)
-    assert 2 * len(df1) == len(df3)
-    assert len(df1.columns) == 7
```

Comment:
Why is this test removed? Should we add an alternative for it to keep the coverage high?

Reply:
This is already removed in the branch for fixing tests, because we don't have the same functionaly for psms objects